include/swift/SILOptimizer/Utils/Local.h

```diff
@@ -282,6 +282,9 @@ class ValueLifetimeAnalysis {
     return LiveBlocks.count(BB) && BB != DefValue->getParent();
   }
 
+  // Checks if there is a dealloc_ref inside the value's liferange.
```

Comment:
I think we have been using the term "live range" instead of life range, no? Also this is an interface so it should be a doxygen comment (i.e. 3 slashes).